src/components/FeedbackForm.vue

```diff
@@ -36,6 +36,11 @@
           @blur="v$.message.$touch()"
           @keydown.enter.prevent="addNewline"
         ></v-textarea>
+        <v-checkbox v-model="gdprAccepted">
```

Comment:
The consent-boolean is called `gdprAccepted`?

I want to suggest to change it into some other name to avoid confusion with `dsgvo_accepted` which is handled [elsewhere](https://github.com/ClockGU/clock-frontend/blob/develop/src/components/gdpr/agreement-components/GdprCardActions.vue).

It is merely a consent for this specific case but it's hard to find a good name... `adminAccesConsent`? This is quite bulky....

Reply:
`accessApproved` would be fitting